ts-rs/tests/issue-70.rs

```diff
@@ -30,15 +32,25 @@ fn issue_70() {
     );
 }
 
+type GenericAlias<A = String, B = String> = HashMap<(A, String), Vec<(B, i32)>>;
+
+#[derive(TS)]
+#[ts(export, export_to = "tests-out/issue_70/")]
+struct Container {
+    a: GenericAlias<Vec<i32>, Vec<String>>,
+    b: GenericAlias,
+}
+
+#[derive(TS)]
+#[ts(export, export_to = "tests-out/issue_70/")]
+struct GenericContainer<A, B = i32> {
+    a: GenericAlias,
+    b: GenericAlias<A, B>,
+    c: GenericAlias<A, GenericAlias<A, B>>,
```

Comment:
This is gonna be a problem as `Record<K, V>` requires that `K extends string | number | symbol`, for that to happen, our `A` generic would need to have that constraint (on top of changing the alias)